lib/rototiller/task/params/env_var.rb

```diff
@@ -127,22 +107,22 @@ def reset
       end
 
       # @private
-      def check
+      def env_value_provided_by_user?
         # its possible that name could not be set
-        ENV.key?(@name) if @name
+        true if (ENV.key?(@name) if @name)
```

Comment:
ok.  largely a nit.  just found it less than readable and i had to go check what the other code did to make sense of this